tests/finance/test_cancel_policy.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+
+import pytz
+
+from unittest import TestCase
+
+from zipline.finance.cancel_policy import NeverCancel, EODCancel
+from zipline.finance.order import Order
+from zipline.gens.sim_engine import (
+    BAR,
+    DAY_END
+)
+
+
+class CancelPolicyTestCase(TestCase):
+
+    def cancel_order(self, order):
+        order.cancel()
+
+    def test_eod_cancel(self):
+
+        order = Order(
+            dt=datetime.datetime(2006, 1, 5, 14, 30, tzinfo=pytz.utc),
```

Comment:
I think we now more commonly use `Timestamp` for setting arbitrary dates, e.g.  `Timestamp('2006-01-05 14:31', tz='UTC')`, but not sure if there is an official policy/standard.